pkg/globalnet/controllers/ipam/constants.go

```diff
@@ -6,10 +6,11 @@ type Operation string
 
 const (
 	handlerResync          = time.Hour * 24
-	submarinerIpamGlobalIp = "submariner.io/globalIp"
-	submarinerIngress      = "SUBMARINER-GN-INGRESS"
-	submarinerEgress       = "SUBMARINER-GN-EGRESS"
-	submarinerMark         = "SUBMARINER-GN-MARK"
+	SubmarinerIpamGlobalIp = "submariner.io/globalIp"
```

Comment:
Seems like keeping it lowerSnake would be more consistent with the parallel vars.

Reply:
Instead of duplicating the constant in e2e tests, I wanted to re-use the same constant (any changes to this annotation in the future will be foolproof). So mainly for this purpose, I modified it to "S"

Reply:
It's better to duplicate, because in that case if somebody changes it and doesn't realize that it's going to break compatibility at least E2E tests will chime in and fail, and then you see that the change being done is changing the E2E tests which at least would be suspicious.

We are following the same principle of replicating the constants to tests.

Reply:
Okay, make sense. Thanks.